### ISSUE TITLE
How to make @keyframes global?

### ISSUE THREAD

**Issue by elliotwaite** [kaisermann/svelte-preprocess#64]
This global style:
```
<style global>
  @keyframes shared-animation {
    from {
      opacity: 0;
    }
    to {
      opacity: 1;
    }
  }
</style>
```
... gets convetered to this:
```
@keyframes svelte-jq3aj2-shared-animation{from{opacity:0}to{opacity:1}}
```
... but I want it to get converted to this:
```
@keyframes shared-animation{from{opacity:0}to{opacity:1}}
```
How do I make the `global` option apply to `@keyframes` as well, so that I can share them with other components?

**Comment by kaisermann** [comment]
Hi @elliotwaite! Currently, the only way to achieve that is to prefix the animation name with a `-global-{name}`. See https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md#1220

> and keyframe declaration IDs are prefixed with -global-

In any case, this should be covered by the `global` attribute and I hope to implement it soon 😁

**Comment by elliotwaite** [comment]
Got it. Sounds good. Thanks for the info!

**Comment by kaisermann** [comment]
Solved by #65
